Dangerfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Is this necessary in every file? I'm just thinking people may be confused by it.

Reply:
For now yes, there's a friendly warning to add it to new files only if users run it again rubocop. it's necessary for future Ruby upgrade. We can remove it if we don't intend to upgrade ruby later to Ruby 3.

Reply:
Let me look some stuff up regarding this. I have used Rubocop in the past but I don't think I've ever had to append a particular comment to each file, that's too much confusion for anyone.